lib/Getopt/Std.pm

```diff
@@ -14,11 +14,11 @@ Getopt::Std - Process single-character switches with switch clustering
     use Getopt::Std;
 
     getopts('oif:');  # -o & -i are boolean flags, -f takes an argument
-		      # Sets $opt_* as a side effect.
-    getopts('oif:', \%opts);  # options as above. Values in %opts
-    getopt('oDI');    # -o, -D & -I take arg.
-                      # Sets $opt_* as a side effect.
-    getopt('oDI', \%opts);    # -o, -D & -I take arg.  Values in %opts
+		      # Sets $opt_* global variables as a side effect
+    getopts('oif:', \my %opts);  # Options as above, values in %opts
```

Comment:
While recommending `\my %opts` is probably better guidance for a Synopsis than a reference to a global variable, I don't see any tests for that syntax in `lib/Getopt/Std.t`.  Can you add unit tests that exercise this syntax?  (I suspect the p.r. will be good to go thereafter).  Thanks.

Reply:
Looks good.  Unless some new objection emerges, let's merge it into blead tomorrow.